src/NewChain/AuxiliaryChainInteract.ts

```diff
@@ -503,7 +507,7 @@ export default class AuxiliaryChainInteract {
       this.anchorOrganizationDeploymentNonce,
     );
     const anchor = await this.deployAnchor(
-      this.originChainId,
+      originChainId || this.originChainId,
```

Comment:
I am not clear on above change. Why it's needed?

Reply:
in case of ropsten, one in constructor will be passed as `ropsten` whereas the method argument will be `3`

Reply:
https://github.com/mosaicdao/mosaic-chains/issues/195